data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/rewards/GetBlockRewards.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.beaconrestapi.handlers.v1.rewards;
+
+import static tech.pegasys.teku.beaconrestapi.BeaconRestApiTypes.PARAMETER_BLOCK_ID;
+import static tech.pegasys.teku.infrastructure.http.HttpStatusCodes.SC_OK;
+import static tech.pegasys.teku.infrastructure.http.RestApiConstants.EXECUTION_OPTIMISTIC;
+import static tech.pegasys.teku.infrastructure.http.RestApiConstants.FINALIZED;
+import static tech.pegasys.teku.infrastructure.http.RestApiConstants.TAG_BEACON;
+import static tech.pegasys.teku.infrastructure.http.RestApiConstants.TAG_REWARDS;
+import static tech.pegasys.teku.infrastructure.json.types.CoreTypes.BOOLEAN_TYPE;
+import static tech.pegasys.teku.infrastructure.json.types.CoreTypes.STRING_TYPE;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.commons.lang3.NotImplementedException;
+import tech.pegasys.teku.api.migrated.BlockRewardData;
+import tech.pegasys.teku.infrastructure.json.types.SerializableTypeDefinition;
+import tech.pegasys.teku.infrastructure.restapi.endpoints.EndpointMetadata;
+import tech.pegasys.teku.infrastructure.restapi.endpoints.RestApiEndpoint;
+import tech.pegasys.teku.infrastructure.restapi.endpoints.RestApiRequest;
+import tech.pegasys.teku.spec.datastructures.metadata.ObjectAndMetaData;
+
+public class GetBlockRewards extends RestApiEndpoint {
+  public static final String ROUTE = "/eth/v1/beacon/rewards/blocks/{block_id}";
+
+  private static final SerializableTypeDefinition<BlockRewardData> DATA_TYPE =
+      SerializableTypeDefinition.object(BlockRewardData.class)
+          .withField("proposer_index", STRING_TYPE, BlockRewardData::getProposerIndex)
+          .withField("total", STRING_TYPE, BlockRewardData::getTotal)
+          .withField("attestations", STRING_TYPE, BlockRewardData::getAttestations)
+          .withField("sync_aggregate", STRING_TYPE, BlockRewardData::getSyncAggregate)
+          .withField("proposer_slashings", STRING_TYPE, BlockRewardData::getProposerSlashings)
+          .withField("attester_slashings", STRING_TYPE, BlockRewardData::getAttesterSlashings)
```

Comment:
this is the bit that will change if we change the object, we should just make it correct now i think...

Reply:
Okay, should I make them all UInt64?

Reply:
tempted to say long, need to know if they can be negative in this instance, im guessing htey cant but would need to look at the spec...